detect-libc returns `glibc` on Linux

hence tries to locale package @parcel/css-linux-x64-glibc, 

```bash
[admin@2445c87ab686 /home/admin]
$node
Welcome to Node.js v16.15.0.
Type ".help" for more information.
> require('detect-libc').familySync()
'glibc'
>
```

Should `@parcel/css-linux-x64-gnu` be renamed as `@parcel/css-linux-x64-glibc`, or both should be published?

Hi, just to be sure, this is related to the change in https://github.com/parcel-bundler/parcel-css/pull/188

that can cause `Cannot find module '../parcel-css.linux-x64-gnu.node` (Node 14 + Yarn 3.2.1 on GitHub Actions) ?

If not I'll open a separate issue.

--- a/scripts/build-npm.js
+++ b/scripts/build-npm.js
@@ -24,6 +24,9 @@ const sysToNodePlatform = {
   darwin: 'darwin',
   windows: 'win32',
 };
+const abiToNodeLibc = {
+  gnu: 'glibc',
+};
 
 let optionalDependencies = {};
 let cliOptionalDependencies = {};
@@ -76,7 +79,7 @@ function buildNode(triple, cpu, os, abi, t) {
   pkg2.os = [os];
   pkg2.cpu = [cpu];
   if (abi) {
-    pkg2.libc = [abi];
+    pkg2.libc = [abiToNodeLibc[abi] || abi];
   }
   pkg2.main = name;
   pkg2.files = [name];
@@ -106,7 +109,7 @@ function buildCLI(triple, cpu, os, abi, t) {
   pkg2.cpu = [cpu];
   pkg2.files = [binary];
   if (abi) {
-    pkg2.libc = [abi];
+    pkg2.libc = [abiToNodeLibc[abi] || abi];
   }
   delete pkg2.main;
   delete pkg2.napi;

